autoplex/auto/rss/flow.py

```diff
@@ -0,0 +1,357 @@
+"""RSS workflows."""
+
+from __future__ import annotations
+
+import os
+from pathlib import Path
+
+import yaml  # type: ignore
+from jobflow import Flow, Maker, Response, job
+
+from autoplex.auto.rss.jobs import do_rss_iterations, initial_rss
+
+current_dir = Path(__file__).absolute().parent
+RSS_CONFIGURATION_DEFAULTS_FILE_PATH = current_dir / "rss_default_configuration.yaml"
+
+
+@job
+class RssMaker(Maker):
+    """
+    Maker to set up and run RSS for exploring and learning potential-energy surfaces (from scratch).
+
+    Parameters
+    ----------
+    name: str
+        Name of the flow.
+    path_to_default_config_parameters: Path | str
+        Path to the default RSS configuration file 'rss_default_configuration.yaml'
+    """
+
+    name: str = "ml-driven rss"
+    path_to_default_config_parameters: Path | str = RSS_CONFIGURATION_DEFAULTS_FILE_PATH
+
+    @job
+    def make(self, config_file: str | None = None, **kwargs):
+        """
+        Make a rss workflow using the specified configuration file and additional keyword arguments.
+
+        Parameters
+        ----------
+        config_file: str | None
+            Path to the configuration file that defines the setup parameters for the whole RSS workflow.
+            If not provided, the default file 'rss_default_configuration.yaml' will be used.
+        **kwargs: dict, optional
```

Comment:
This can go under the header of Keyword Arguments in doc-strings, will make it consistent to other parts of code. See here , this is how it will be then rendered in API reference. Now I am not sure it will even get rendered properly. 

https://autoatml.github.io/autoplex/reference/autoplex.fitting.common.utils.jace_fitting.html#autoplex.fitting.common.utils.jace_fitting

Reply:
Sounds good!